src/main/java/lotto/model/LottoSeller.java

```diff
@@ -3,25 +3,25 @@
 public class LottoSeller {
     private static int LOTTO_COST = 1000;
```

Comment:
```suggestion
    private static final int LOTTO_COST = 1000;
```

상수를 수정하지 못하도록 final을 붙이는게 좋아요~